frontend/server/src/test/resources/config_invalid_port.properties

```diff
@@ -0,0 +1,5 @@
+inference_address=https://127.0.0.1:65536
+management_address=unix:/tmp/management.sock
+# model_server_home=../..
+model_store=../modelarchive/src/test/resources/models
+enable_envvars_config=true
```

Comment:
Do we need `inference_address` to be `true`? The configuration seems to be set in the properties file itself.